src/sentry/south_migrations/0464_groupenvironment_foreignkeys.py

```diff
@@ -0,0 +1,1314 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        pass
```

Comment:
We just need this as a dummy migration since I changed the bare ids to foreign keys